Unused import of image/draw after latest changes

In e1dbfaf9f0d2f708ae09ba3fee2f9b986302cf17, code in [/v3.3/glfw/input.go](https://github.com/go-gl/glfw/blob/master/v3.3/glfw/input.go) and [/v3.3/glfw/window.go](https://github.com/go-gl/glfw/blob/master/v3.3/glfw/window.go) that used `image/draw` was removed. However, the import of `image/draw` is still there, leading builds to fail:

```
$ go build
# github.com/go-gl/glfw/v3.3/glfw
..\..\..\..\go\pkg\mod\github.com\go-gl\glfw\v3.3\glfw@v0.0.0-20231031140856-0d7f346fb12e\input.go:23:2: "image/draw" imported and not used
..\..\..\..\go\pkg\mod\github.com\go-gl\glfw\v3.3\glfw@v0.0.0-20231031140856-0d7f346fb12e\window.go:19:2: "image/draw" imported and not used
```

--- a/v3.3/glfw/input.go
+++ b/v3.3/glfw/input.go
@@ -20,7 +20,6 @@ import "C"
 
 import (
 	"image"
-	"image/draw"
 	"unsafe"
 )
 

--- a/v3.3/glfw/window.go
+++ b/v3.3/glfw/window.go
@@ -16,7 +16,6 @@ import "C"
 
 import (
 	"image"
-	"image/draw"
 	"sync"
 	"unsafe"
 )

